index.js

```diff
@@ -118,20 +128,23 @@ const parseArgs = (
         if (arg.length > 2) {
           for (let i = 2; i < arg.length; i++) {
             const short = StringPrototypeCharAt(arg, i);
+            // Case of `-o=foo`:
```

Comment:
I do not consider this a short option and its value in the way you are assuming. The `=` is for long options not for short options. The four possible forms for specifying the expected value are:

```
-v VVV
-vVVV // if we support it
--value VVV
--value=VVV
```

(Or on the end of a combined short group.)

Reply:
Expanding the = to an argument seems off to me, so I think think that two options are either throw a parsing error, or treat it the same as:

-v foo

What does POSIX do?

Reply:
I learnt something rereading the Utility Conventions. `=` is not a valid short option name, so we are into undefined behaviour. (I thought I saw something about "printable" somewhere, didn't find that just now.)

I have not checked `getopt` to see how it deals with this.

> Guideline 3:
> Each option name should be a single alphanumeric character (the `alnum` character classification) from the portable character set. 

https://pubs.opengroup.org/onlinepubs/9699919799.2016edition/basedefs/V1_chap12.html#tag_12_01